docs/source/main_classes/deepspeed.rst

```diff
@@ -1555,9 +1574,60 @@ stress on ``tensor([1.])``, or if you get an error where it says the parameter i
 larger multi-dimensional shape, this means that the parameter is partitioned and what you see is a ZeRO-3 placeholder.
 
 
+
+
+Filing Issues
+=======================================================================================================================
+
+Here is how to file an Issue so that we could quickly get to the bottom of the Issue and help you to unblock your work.
+
+In your report please always include:
+
+1. the full Deepspeed config file in the report
+
+2. either the command line arguments if you were using the :class:`~transformers.Trainer` or
+   :class:`~transformers.TrainingArguments` arguments if you were scripting the Trainer setup yourself. Please do not
+   dump the :class:`~transformers.TrainingArguments` as it has dozens of entries that are irrelevant.
+
+3. Output of:
+
+.. code-block:: bash
+
+    python -c 'import torch; print(f"torch: {torch.__version__}")'
+    python -c 'import transformers; print(f"transformers: {transformers.__version__}")'
+    python -c 'import deepspeed; print(f"deepspeed: {deepspeed.__version__}")'
+
+4. If possible include a link to a Google Colab notebook that we can reproduce the problem with. You can use this
+   `notebook <https://github.com/stas00/porting/blob/master/transformers/deepspeed/DeepSpeed_on_colab_CLI.ipynb>`__ as
+   a starting point.
+
+5. Unless it's impossible please always use a standard dataset that we can use and not something custom.
+
+6. If possible try to use one of the existing `examples
+   <https://github.com/huggingface/transformers/tree/master/examples/pytorch>`__ to reproduce the problem with.
+
+Things to consider:
+
+* Deepspeed is often not the cause of the problem.
+
+   Some of the filed issues proved to be Deepspeed-unrelated. That is once Deepspeed was removed from the setup, the
+   problem was still there.
+
+   Therefore, if it's not absolutely obvious it's a DeepSpeed-related problem, as in you can see that there is an
+   exception and you can see that DeepSpeed modules are involved, first re-test your setup without DeepSpeed in it. And
+   only if the problem persists then do mentioned Deepspeed and supply all the required details.
```

Comment:
Indent is weird here.

Reply:
That's so the nested paragraphs of the item get aligned with the item - if this is not done we get this:

![snapshot_2](https://user-images.githubusercontent.com/10676103/123139092-acdf2a00-d40a-11eb-8a1f-38de446c4c66.png)

I guess I need to make it indent 4 and not 3, correct?

Reply:
changed it to 4 - is it ok now?

Reply:
Still weird but if sphinx wants it, we don't have a choice.

Reply:
We have a lot of the same in testing.rst - it basically has no way of telling that the item continues in the next paras unless you indent it. Same with code sections inside items.

It's the same as markdown.